src/js/utils/clock.ts

```diff
@@ -1,6 +1,7 @@
 import { now as getTime } from 'utils/date';
+import { GenericObject } from '../types/generic.type';
 
-const performance = window.performance || {
+const performance: Performance | GenericObject = window.performance || {
```

Comment:
should we type `{ timing: GenericObject}` instead for the second half of the union?

Reply:
Explained earlier https://github.com/jwplayer/jwplayer/pull/3622#discussion_r384792807